test/files/run/t7747-repl.check

```diff
@@ -236,45 +236,39 @@ package $line10 {
       super.<init>;
       ()
     };
-    sealed class $iw extends _root_.java.io.Serializable {
-      def <init>() = {
-        super.<init>;
-        ()
-      };
-      import $line7.$read.INSTANCE.$iw.$iw.BippyBups;
-      import $line7.$read.INSTANCE.$iw.$iw.BippyBups;
-      import $line8.$read.INSTANCE.$iw.$iw.PuppyPups;
-      import $line8.$read.INSTANCE.$iw.$iw.PuppyPups;
-      import $line9.$read.INSTANCE.$iw.$iw.Bingo;
-      import $line9.$read.INSTANCE.$iw.$iw.Bingo;
-      sealed class $iw extends _root_.java.io.Serializable {
-        def <init>() = {
-          super.<init>;
-          ()
-        };
-        val res3 = List(BippyBups, PuppyPups, Bingo)
-      };
-      val $iw = new $iw.<init>
-    };
-    val $iw = new $iw.<init>
+    import _root_.scala.tools.nsc.interpreter.$u007B$u007B;
+    import _root_.scala.tools.nsc.interpreter.$u007B$u007B;
```

Comment:
Not sure what leads to this, but we could collapse sequences of these into a single import.